test/extended/util/test.go

```diff
@@ -394,7 +394,8 @@ var (
 
 			`should idle the service and DeploymentConfig properly`, // idling with a single service and DeploymentConfig [Conformance]
 
-			`\[Feature:Volumes\]`, // storage team to investigate it post-rebase
+			`\[Feature:Volumes\]`,       // storage team to investigate it post-rebase
+			`\[Testpattern: Dynamic PV`, // storage team to investigate it post-rebase. @hekumar
```

Comment:
It's CSI what's broken, I'd prefer `[Driver: csi-hostpath`. Non-CSI AWS EBS test pass and it's good to have them in.

This can be changed post-rebase, of course.